tests/unit/task_system_test.c

```diff
@@ -95,6 +96,11 @@ CTEST_SETUP(task_system)
    // Do minimal IO config setup, using default IO values.
    master_config master_cfg;
    config_set_defaults(&master_cfg);
+
+   // Expected args to parse: --num-normal-bg-threads, --num-memtable-bg-threads
```

Comment:
DO not have test code take arguments. Rework the body to have a separate test sub-case which will deal with specialized arguments.

Reply:
Reworked unit-test cases accordingly.